meta/entity/draft-1.json

```diff
@@ -17,7 +29,8 @@
         }
       },
       "required": [
-        "aliases"
+        "aliases",
```

Comment:
If we're still using `aliases` maybe it should become capital like the rest of them. And maybe renamed to Synonyms?

Reply:
fixed 1c2240d9e49e8cbd4343cf2cc8ac551f2af9bf92